bibliofil_lists.module

```diff
@@ -0,0 +1,293 @@
+<?php
+
+/**
+ * @file
+ * Bibliofil lists module.
+ */
+
+define('DYNPATH', '/cgi-bin/m2dyn.htmc');
+define('WEBLISTERPATH', '/cgi-bin/rest_service/web_lister/1.0/data/');
```

Comment:
Please prefix the constants with module name.
https://www.drupal.org/docs/develop/standards/coding-standards#s-constants